src/plugins/Sharesite/webui/EditToadlet.java

```diff
@@ -114,62 +126,76 @@ private void addNodes( HTMLNode form, String name, String desc, String text, Str
 		vals = new String[] { "textInput", "20", "80", "font-size: medium;" };
 		textDiv.addChild("textarea", attrs, vals, text);
 
+		HTMLNode insertHour = editBox.content.addChild("p");
```

Comment:
I think it’s ill-advised to call an HTMLNode “insertHour” while the actualy insert hour is called “iHour” which looks like a variable with the Hungarian prefix for an integer but is actually a `String`. That’s, like, maximum surprise.